test/integration/util.js

```diff
@@ -57,6 +57,6 @@ function project (coordinates) {
     const wm = projectToWebMercator(coordinates);
     return {
         x: mapSize * (0.5 + wm.x / WM_2R),
-        y: mapSize * (0.5 - wm.y / WM_2R + 0.03 /* offset */)
+        y: mapSize * (0.5 - wm.y / WM_2R)
```

Comment:
Could you explain this since you introduced the magic `0.03`? Why was it added? The addition was done on: https://github.com/CartoDB/carto-vl/commit/7b46dee5273f30cc48216b9f1c04f180f2f7e834

Reply:
This changed was required to match the bounding box in the simulated mouse events. I have updated the bounding box in the `carto.Map` so this is not required anymore.

Reply:
I don't really understand it, but if you are sure... :+1: